src/app/workflow/workflow.component.ts

```diff
@@ -87,8 +87,9 @@ export class WorkflowComponent extends Entry implements AfterViewInit {
   pubUnpubMessage: string;
   @Input() user;
 
-  constructor(private dockstoreService: DockstoreService, dateService: DateService, private bioschemaService: BioschemaService, private refreshService: RefreshService,
-    private workflowsService: WorkflowsService, trackLoginService: TrackLoginService, providerService: ProviderService,
+  constructor(private dockstoreService: DockstoreService, dateService: DateService, private bioschemaService: BioschemaService,
```

Comment:
Reminder that you'll need to rebase/merge develop for the current develop's formatting changes